### DISCUSSION TITLE
Rollback to older snapshot

### DISCUSSION THREAD

**Discussion by jhein05** [hassio-addons/addon-zwavejs2mqtt#227] [General]
Hi,

quick question: If I rollback to an older snapshot of the zwavejs2mqtt addon, will the devices that I added since creating that snapshot still be there or do I have to add them again? 

I have problems doing an OTA firmware update to a switch, but have done multiple updates to that switch model before and I would like to investigate if it is a problem with the switch or zwavejs2mqtt.

Thanks,
Julian

**Comment by frenck** [comment]
Devices are bound to the stick, not the software.
